src/sql/workbench/services/objectExplorer/browser/dragAndDropController.ts

```diff
@@ -86,6 +86,9 @@ export class ServerTreeDragAndDrop implements IDragAndDrop {
 			escapedSchema = this.escapeString(element.metadata.schema);
 			escapedName = this.escapeString(element.metadata.name);
 			finalString = escapedSchema ? `[${escapedSchema}].[${escapedName}]` : `[${escapedName}]`;
+			if (element.nodeTypeId === 'Function') {
```

Comment:
it seems weird to have so many special-cases for "Function" .  Is there a way to do this more generically?